packages/schematics/angular/e2e/files/src/app.po.ts.template

```diff
@@ -1,11 +1,11 @@
 import { browser, by, element } from 'protractor';
 
 export class AppPage {
-  navigateTo(): Promise<unknown> {
+  async navigateTo(): Promise<unknown> {
```

Comment:
FWIW, you can also use `PromiseLike<...>` here and not have to cast below.
Do you know if there are any downsides to using `PromiseLike`?

Reply:
`PromiseLike` is mainly useful when dealing with alternative implementations of promises (bluebird, etc.).  The only real downside is that it won't be accepted for any consumer that expects a `Promise` (e.g., `function myFunc(p: Promise<unknown>)`).

Reply:
Also, no reason for `async` here since you are not using `await`.